about.html

```diff
@@ -0,0 +1,181 @@
+                    </div>
+
+                    <div class="cs-africa non">
+
+                    </div>
+
+                    <a href="index.html"><img src="img/CS-logo.png" class="page-logo" alt="Page Logo"></a>
+                    <ul class="nav-items">
+                        <li class="nav-item"><a href="about.html">About</a></li>
+                        <li class="nav-item"><a href="#">Program</a></li>
+                        <li class="nav-item"><a href="#">Join</a></li>
+                        <li class="nav-item"><a href="#">Sponsor</a></li>
+                        <li class="nav-item"><a href="#">News</a></li>
+                        <a href="index.html" class="nonee">
+                            <i class="bi bi-house-door-fill side-home-icon"></i> 
+                        </a>
+                    </ul>
+                    <button class="nav-button">CS Campaign</button>
+                </div>
+            </div>
+        </nav>
+
+        <div class="about-header-content">
+            <div class="page-nav">
+                <ul>
+                    <li>
+                        <a href="index.html">
+                            <i class="bi bi-house-door-fill home-icon"></i> 
+                        </a>
+                    </li>
+                    <i class="bi bi-chevron-right"></i>
+                    <li>
+                        <a href="about.html">About</a>
+                    </li>
+                </ul>
+            </div>
+
+            <div class="about-content">
+                <h1>DIGITAL TRANSFORMATION <br class="none"> AFRICAN SUBMIT 2023</h1>
+                <hr class="hr-line none">
+            </div>
+            <div class="header-content">
+                <p class="description">
+                    The digital landscape is constantly evolving, and businesses and individuals must adapt to stay relevant.
+                    This web summit aims to explore the concept of digital transformation and how organizations can leverage technology to thrive in the future.
+                    It will bring together industry experts, leaders, and innovators to share insights, strategies, and best practices for navigating the digital landscape successfully.
+                    <span class="none">
+                    So if you are active and engaged in the worlds of open content and knowledge-free software advocates, Wikipedians, Open
+                    knowledge, galleries, museums, archives, governemnt and foundations, lawyers and activists-we hope you'll
+                    consider joining us this year to build a stronger, more vibrant commons together.</span>
+                </p>
+            </div>
+            <h6 class="py-3 mt-3 outer">For any further questions or inquires about the digital transformation 2023 ,Email us @</h6>
+            <p class="header-email mb-3">ccsummit2023@transformation.org <i class="bi bi-chevron-right"></i></p>
+        </div>
+    </section>
```

Comment:
- [ ] Kindly fix the header section of the `about` section, so it shows correctly, because the user can not see it entirely 👍.

Your design 
![Capture1](https://github.com/Edgahkipkemoi/capstone/assets/80458199/6ee35cd1-b1b4-4ab9-ae8d-ec36672a445f)


Cindy design
![Capture](https://github.com/Edgahkipkemoi/capstone/assets/80458199/123744df-eac7-47f5-89f2-7bed43b52e3e)